backend/svc/config.go

```diff
@@ -0,0 +1,17 @@
+package svc
+
+import (
+	"go.autokitteh.dev/autokitteh/internal/backend/config"
+)
+
+type Config = config.Config
```

Comment:
I don't like the liberal use we do with type aliases, they are intended for code refactoring.

Reply:
this is only for exposing internal stuff through this specific package. I don't think I am using aliases anywhere else tbh